lib/instrumentation/mongodb/common.js

```diff
@@ -272,6 +279,11 @@ function getParametersFromTopology(conf, database) {
     ;[{ host, port }] = conf.s.options.servers
   }
 
+  // hosts is an array but we will always pull the first for consistency
+  if (conf?.s?.options?.hosts?.length) {
+    ;[{ host, port }] = conf.s.options.hosts
+  }
```

Comment:
I'm pretty sure it being an array is for sharding. Using only the first entry, are we creating any issues for the user?

Reply:
it's the pattern we have [above](https://github.com/newrelic/node-newrelic/pull/2085/files#diff-7e1a46c38f4d81d5e6e9ea3e8c98bfa8495bcc5d64900262ead8d98391464143R279) and replicating logic for consistency